.circleci/config.yml

```diff
@@ -2,8 +2,8 @@ version: 2.1
 orbs:
   slack: circleci/slack@3.4.2
   cla-end-to-end-tests: ministryofjustice/cla-end-to-end-tests@volatile
-  aws-cli: circleci/aws-cli@4.0.0 # use v4 of this orb
-  aws-ecr: circleci/aws-ecr@8.2.1 # this orb doesn't support OIDC v2, so we use aws-cli to authenticate
```

Comment:
Just a thought - being less precise with these dependencies might be preferable e.g. `9.0`- we found 9.0.1 of this was [suddenly removed](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1282). And we probably want to adopt patch releases of these orbs by default

Reply:
Thanks @davidread, that's a good point. If releases are going to be removed lets make sure we're using the latest point release.